### PR TITLE
Misc housekeeping - webpack dev vs prod and changing one action from GET to POST

### PR DESCRIPTION
**Pull request by paulkva**
Per discussion between @Dani21 and me over the past day or so. Setting it up this way speeds up running in local/dev mode significantly.

### CODE REVIEW

**Comment by paulkva** [comment]
Note with the addition of one more housekeeping item, we need to coordinate the deployment of this PR with deployment of the API repo.

File: package.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
   "version": "3.6.0",
   "description": "map",
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack --mode=production",
-    "start": "cross-env NODE_ENV=development webpack serve",
+  	"build": "echo use 'npm run prodbuild' or 'npm run devbuild'",
```
Comment (corywright) [inline package.json:6]: Looks like an extra indentation here.

File: package.json [hunk 2]
```diff
@@ -3,8 +3,10 @@
...
+    "prodbuild": "cross-env NODE_ENV=production webpack --mode=production",
```
Comment (corywright) [inline package.json:7]: The instructions in `documents/deployment.md` should also be updated to match this change.

File: package.json (hunk 1 above)
Comment (paulkva) [reply package.json]: fixed ✔️

File: package.json (hunk 2 above)
Comment (paulkva) [reply package.json:7]: fixed ✔️

**Review by corywright — APPROVED** [review]
Will wait for @Dani21 to approve before merging.

**Review by Dani21 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -7,15 +7,25 @@ Source code for the web portion of [supercharge.info](https://supercharge.info).
...
+
+`npm install`
+`npm run start`
+
+http://localhost:9090/
+
+### Running an open development server (choose port and comma-separated list of allowed hostnames)
+
+`npm install`
+`npm run start -- --env open --env port=9191 --env hosts=mydevsite,mydevsite.dev.supercharge.info`
```
[the remark this answers is not in the document]
Comment (Dani21) [reply README.md:22]: Actually I was using an older version! I see why this should work now

**Comment by paulkva** [comment]
I'm going to merge this and then submit a separate PR for more housekeeping/fixes and WayBack enhancements.

**Pull request merged**